lib/src/signed_video_h26x_common.c

```diff
@@ -843,16 +843,69 @@ update_gop_hash(void *crypto_handle, gop_info_t *gop_info)
 
   return status;
 }
+
+svrc_t
+initialize_and_update_gop_level_hash(signed_video_t *self, uint8_t *hash_list, int idx)
+{
+
+  svrc_t status = SV_UNKNOWN_FAILURE;
+
+  SV_TRY()
+    // Initialize the hashing process for the fallback GOP-level hash
+    SV_THROW(openssl_init_hash(self->crypto_handle, true));
+    SV_THROW(openssl_update_hash(self->crypto_handle, hash_list, idx, true));
+
+  SV_CATCH()
+  SV_DONE(status)
+
+  return status;
+}
+
+svrc_t
+update_fallback_gop_hash(signed_video_t *self, const uint8_t *nalu_hash)
```

Comment:
Wrapping `openssl_update_hash(...)` in a function like this adds no value. Remove.